vscode/features.json5

```
@@ -0,0 +1,102 @@
+{
+    "editors": ["vscode", "jetbrains"],
+    "features": [
+        {
+        "name": "NotebookBasedChatUI",
+        "description": "new chat UI that resembles a notebook instead of a chat session",
+        "productLine": "all",
+        "documentation": [],
+        "editors": {
+          "vscode": {
+            "status": "stable",
+          }
+        },
+        tags: ['chat']
```

```suggestion
        tags: ["chat"]
```

Nit, don't needlessly vary choice of quote.